pkg/registry/apis/dashboard/register.go

```diff
@@ -167,7 +169,8 @@ func (b *DashboardsAPIBuilder) GetGroupVersions() []schema.GroupVersion {
 	return []schema.GroupVersion{
 		dashv1.DashboardResourceInfo.GroupVersion(),
 		dashv0.DashboardResourceInfo.GroupVersion(),
-		dashv2.DashboardResourceInfo.GroupVersion(),
+		dashv2alpha2.DashboardResourceInfo.GroupVersion(),
```

Comment:
Same here – I assume that `v2alpha2` takes precedence over `v2alpha1` on purpose?